modules/installation-aws-ami-stream-metadata.adoc

```diff
@@ -0,0 +1,31 @@
+//TODO: Add the module include to the following assemblies
+//TODO: Create related modules for OpenStack (QCOW2 and Bare Metal ISO)
+
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+
+[id="installation-aws-ami-stream-metadata_{context}"]
+= Printing {op-system} AMIs for AWS zones by using stream metadata
+
+In {product-title}, _stream metadata_ provides standardized metadata about {op-system} in the JSON format and injects the metadata into the cluster. Stream metadata is a stable format that supports multiple architectures and is intended to be self-documenting for maintaining automation.
+
+Stream metadata is available in a Go library that contains example code for the software. The official `stream-metadata-go` library and example code is available at https://github.com/coreos/stream-metadata-go.
+
+You can use the `coreos print-stream-json` sub-command of `openshift-install` to access information about the boot images in the stream metadata format. This command provides a method for printing stream metadata in a scriptable, machine-readable format. For user-provisioned installations, the `openshift-install` binary contains pinned versions of {op-system} boot images, such as an AWS AMI.
```

Comment:
I'd invert these and put this first, and then recommend that if you want to parse it from a Go program, use the library. 

Then, also mention that parsing it via `jq` or any other tooling that handles JSON (e.g. any other programming language's JSON library) is also fine.